docs/UserGuide.md

```diff
@@ -595,5 +629,12 @@ Action     | Format, Examples
 **[Sort](#sort-deliveries-sort-d)**   | `sort -d so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -d so/a sb/c`
 **[Upcoming](#upcoming-deliveries-upcoming)** | `upcoming aft/START_DATE bef/END_DATE`<br> e.g., `upcoming aft/19-12-2022 08:00 bef/18-06-2023 17:00`
 
+## Glossary
+
+* **Supplier Contact**: A record containing information about a supplier, including name, company, contact number, email, associated products and tags.
+* **Java**: 
+
+NEED UPDATE TERMS AT THE END!! CAN LMK IF YOU HAVE ANY IN THE PR OR SMTH
+
```

Comment:
CLI 
parameter
prefix 
Command.
Index
jar 

are some terms i think we should define